persistence/src/commonMain/kotlin/com/wire/kalium/persistence/model/AuthSessionEntity.kt

```diff
@@ -13,14 +13,22 @@ data class ServerConfigEntity(
         val teams: String,
         val website: String,
         val title: String,
-        val isOnPremises: Boolean
+        val isOnPremises: Boolean,
+        val proxy: Proxy?
+
     )
 
     data class MetaData(
         val federation: Boolean,
         val apiVersion: Int,
         val domain: String?
     )
+
+    data class Proxy(
+        val needsAuthentication: Boolean?,
+        val apiProxy: String?,
+        val port: Int?
```

Comment:
Are any of these properties actually optional?

Reply:
Whenever there is proxy object no it should be not optional, so yeah I will update it to not be nullable